From 2227ac005de1d004608b163ac1cc37cf1ffb86b1 Mon Sep 17 00:00:00 2001
From: Benjamin Tissoires <benjamin.tissoires@gmail.com>
Date: Tue, 7 Nov 2017 14:58:32 +0100
Subject: [PATCH libwacom] data: Add Dell Canvas 27

The touchscreen part is not from Wacom, so we need to
teach libwacom and libinput that both are part of the
same device group for them to do touch arbitration.

Signed-off-by: Benjamin Tissoires <benjamin.tissoires@gmail.com>
Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
The 0x80a Eraser is a dummy to avoid crashes in g-s-d versions
that cannot handle eraser-less devices (pre 3.26).

 data/dell-canvas-27.tablet | 37 +++++++++++++++++++++++++++++++++++++
 1 file changed, 37 insertions(+)
 create mode 100644 data/dell-canvas-27.tablet

diff --git a/data/dell-canvas-27.tablet b/data/dell-canvas-27.tablet
new file mode 100644
index 0000000..1c82390
--- /dev/null
+++ b/data/dell-canvas-27.tablet
@@ -0,0 +1,37 @@
+# Dell
+# Canvas 27
+#
+#
+#
+#          *-----------------------*
+#          |                       |
+#          |                       |
+#          |                       |
+#          |                       |
+#          |                       |
+#          |        DISPLAY        |
+#          |                       |
+#          |                       |
+#          |                       |
+#          |                       |
+#          |                       |
+#          *-----------------------*
+#
+
+[Device]
+Name=Dell Canvas 27
+Class=Cintiq
+DeviceMatch=usb:056a:4200
+PairedID=usb:2575:0204
+Width=23
+Height=13
+# No pad buttons, so no layout
+Styli=0xffffd;0x80a;
+IntegratedIn=Display
+
+[Features]
+Stylus=true
+Reversible=false
+Touch=true
+Ring=false
+Buttons=0
--
2.13.6
